Source/WebKit/WPEPlatform/wpe/wayland/WPEViewWayland.cpp

```diff
@@ -93,13 +116,9 @@ struct DMABufFeedback {
         Data* data { nullptr };
     };
 
-    static std::unique_ptr<DMABufFeedback> create(unsigned size, int fd)
+    static std::unique_ptr<DMABufFeedback> create()
     {
-        auto* data = static_cast<FormatTable::Data*>(mmap(nullptr, size, PROT_READ, MAP_PRIVATE, fd, 0));
-        if (data == MAP_FAILED)
-            return nullptr;
-
-        return makeUnique<DMABufFeedback>(FormatTable(size, data));
+        return makeUnique<DMABufFeedback>(FormatTable());
```

Comment:
I guess we can now change the DMABufFeedback constructor to not receive a FormatTable, since we always want an empty one now.

Reply:
Good idea.